quri_sdk_notebooks/examples/0_quri-algo-vm/1_spe/1_spe.ipynb

```diff
@@ -136,30 +136,63 @@
+          ]
+        }
+      ],
       "source": [
         "from quri_parts.circuit.utils.circuit_drawer import draw_circuit\n",
         "from quri_algo.circuit.time_evolution.trotter_time_evo import (\n",
         "    TrotterControlledTimeEvolutionCircuitFactory,\n",
         ")\n",
         "\n",
         "trotter_concotrolled_time_evo_circuit_factory = (\n",
-        "    TrotterControlledTimeEvolutionCircuitFactory(hamiltonian_input, n_trotter=50)\n",
+        "    TrotterControlledTimeEvolutionCircuitFactory(hamiltonian_input, n_trotter=30)\n",
         ")\n",
         "\n",
-        "# Time-evolution with t=10\n",
+        "# Time-evolution with t=1\n",
         "c_time_evo = trotter_concotrolled_time_evo_circuit_factory(evolution_time=1)\n",
         "\n",
-        "# draw_circuit(c_time_evo, line_length=10000)"
+        "draw_circuit(c_time_evo, line_length=10000)"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "As seen above, the time-evolution is composed of a series of Pauli rotations. Since we chose a single Trotter step and since it is a controlled time-evolution, each term in the Hamiltonian roughly results in two Pauli rotations. We represent these Pauli rotations as single quantum gates, however to run them on a real device, in most cases, they need to first be transpiled to a series of two-qubit gates and single qubit rotations. Later in this notebook, we will show how this is done to conform to the [STAR architecture](https://journals.aps.org/prxquantum/abstract/10.1103/PRXQuantum.5.010337)."
+        "As seen above, the time-evolution is composed of a series of Pauli rotations. Since since it is a controlled time-evolution, each term in the Hamiltonian results in two Pauli rotations, i.e.\n",
```

Comment:
```suggestion
        "As seen above, the time-evolution is composed of a series of Pauli rotations. Since it is a controlled time-evolution, each term in the Hamiltonian results in two Pauli rotations, i.e.\n",
```